lineman/app/components/navbar/navbar.coffee

```diff
@@ -11,6 +11,8 @@ angular.module('loomioApp').directive 'navbar', ->
 
     $scope.hostName = parser.hostname
 
+    $scope.isLoggedIn = ->
+      AbilityService.isLoggedIn()
```

Comment:
I wondered if this should be `AppConfig.isLoggedIn()`, but it felt more like an ability of the user than a configuration thing.